.github/PULL_REQUEST_TEMPLATE/docs.md

```diff
@@ -0,0 +1,37 @@
+## Description
+
+Closes: #XXXX
+
+<!-- Add a description of the changes that this PR introduces and the files that
+are the most critical to review. -->
+
+<!-- Pull requests that sit inactive for longer than 30 days will be closed.  -->
+---
+
+### Author Checklist
+
+*All items are required. Please add a note to the item if the item is not applicable and
+please add links to any relevant follow up issues.*
+
+I have...
+
+- [ ] included the correct `docs:` prefix in the PR title
+- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
+- [ ] provided a link to the relevant issue or specification
+- [ ] reviewed "Files changed" and left comments if necessary
+- [ ] confirmed all CI checks have passed
+
+### Reviewers Checklist
+
+*All items are required. Please add a note if the item is not applicable and please add
+your handle next to the items reviewed if you only reviewed selected items.*
+
+I have...
+
+- [ ] confirmed the correct `docs:` prefix in the PR title
+- [ ] confirmed all author checklist items have been addressed 
+- [ ] confirmed that this PR only changes documentation
+- [ ] reviewed content for consistency
+- [ ] reviewed content for thoroughness
+- [ ] reviewed content for spelling and grammar
+- [ ] tested instructions (if applicable)
```

Comment:
Can you elaborate on what is meant by _Tested instructions_? Does that imply there are user flows and previously established acceptance criteria that should be tested against?

Reply:
If the documentation updates a tutorial that contains CLI commands, then those instructions should be tested. For example, changes to https://hub.cosmos.network/main/getting-started/installation.html should be tested to see if the steps described there still work.